src/components/BToggle/BToggle.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+
+  <div class="toggle-container">
+    <label class="toggle">
+      <input type="checkbox" :disabled="disabled" :checked="checked">
+
+      <span class="slider"><label :class='labelClass'>{{this.label}}</label></span>
+    </label>
+  </div>
+  
+</template>
+
+<script>
+export default {
+  name: 'b-toggle',
+  props: {
+    id:{
+      type: String,
+      required: true
+    },
+    label:{
+      type: String,
+      required: true
+    },
+    required:{
+      type: Boolean,
+      required: false
+    },
+    disabled:{
+      type: Boolean,
+      required: false
+    },
+    checked:{
+      type: Boolean,
+      required: false
+    }
+  },
+  data () {
+    return {}
+  },
+  computed: {
+    labelClass: function () {
```

Comment:
This is where I think a scoped style doesn't do us justice. All the inputs have the label style done differently when I think they should all probably look the same, right?

Reply:
what do you mean?  My disabled state sets the color and opacity the same is the other disabled states?

Reply:
Right, I think Steven is saying that since all the components use the same style, they should in fact use the same css instead of each scoped style being done the same.